annotate/module.ily

```diff
@@ -74,7 +74,8 @@
       (props (context-mod->props properties))
       ;; retrieve a pair with containing directory and input file
       (input-file (string-split (car (ly:input-file-line-char-column (*location*))) #\/ ))
-      (ctx (list-tail input-file (- (length input-file) 2)))
+      (ctx (list-tail input-file (- (length input-file)
+                                    (if (> (length input-file) 1) 2 1))))
```

Comment:
I'm not at my LilyPond computer ATM, but I do doubt this is possible. Either there would be some side-effects you didn'T notice yet or - if the change does *not* matter - then the code should be somehow flawed in the first place. I'll have to investigate closer what it should actually do.